alpha/lib/model/FileSync.php

```diff
@@ -282,7 +282,18 @@ public function getStatusAsString()
 	{
 		return (isset($this->statusMap[$this->getStatus()])) ? $this->statusMap[$this->getStatus()] : "Unknown";
 	}
-	
+
+	protected function getServeUrl($entryId)
+	{
+		$dynamicAttributes = DeliveryProfileDynamicAttributes::init($this->getDc(), $entryId, PlaybackProtocol::HTTP, infraRequestUtils::getProtocol());
```

Comment:
sending - PlaybackProtocol::HTTP, infraRequestUtils::getProtocol() is not needed use default